pkg/controllers/multiclusterservice/mcs_controller.go

```diff
@@ -95,8 +107,13 @@ func (c *MCSController) Reconcile(ctx context.Context, req controllerruntime.Req
 			return
 		}
 		_ = c.updateMCSStatus(mcs, metav1.ConditionTrue, "ServiceAppliedSucceed", "Service is propagated to target clusters.")
+		c.EventRecorder.Eventf(mcs, corev1.EventTypeWarning, events.EventReasonSyncWorkSucceed, "Service is propagated to target clusters.")
 	}()
 
+	if err = c.buildResourceInformers(); err != nil {
```

Comment:
I am not sure whether this implementation is ok. I use this way: https://github.com/karmada-io/karmada/pull/4356/files#diff-673e6c422a4a07b894d916c2513cdf4785264a58daa44a24d344ab9ae9b00d8bR173